src/packages/templating/components/templating-item-menu/templating-item-menu.element.ts

```diff
@@ -59,44 +59,50 @@ export class UmbTemplatingInsertMenuElement extends UmbLitElement {
 
 	async #openTemplatingItemPickerModal() {
 		const itemPickerContext = this.#modalContext?.open(this, UMB_TEMPLATING_ITEM_PICKER_MODAL);
-		await itemPickerContext?.onSubmit();
-
-		const value = itemPickerContext?.getValue();
-		if (!value) return;
-
-		this.determineInsertValue(value);
+		await itemPickerContext
+			?.onSubmit()
+			.then((value) => {
+				if (value) this.determineInsertValue(value);
+			})
+			.catch(() => undefined);
```

Comment:
I would recommend you doing it like this as it reads cleaner:

```suggestion
		const result = await itemPickerContext?.onSubmit().catch(() => undefined);
		
		if (result === undefined) return;
		
		const value = itemPickerContext?.getValue();
		
		if (!value) return;
		
		this.determineInsertValue(value);
```